app/api/v2/common/responses.py

```diff
@@ -18,6 +18,7 @@
 class Success(GenericModel, Generic[T]):
     status: Literal["success"]
     data: T
+    meta: Optional[dict[str, Any]]
```

Comment:
success() changes this to dict below
```suggestion
    meta: dict[str, Any]
```